tasks/3_is-boolean/is-boolean.js

```diff
@@ -0,0 +1,5 @@
+export const isBoolean = (value) => {
+  if (typeof value === 'boolean') {
```

Comment:
you don't need `if` here as typeof value === 'boolean' will already return what you need

Reply:
it will not work for "new Boolean(true)" because typeof Boolean is object :D